.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 4
   - 6
```

Comment:
not a necessary...while do you mind adding node 8 and 10 for test here? Thanks, we can revert it if tests fail and i will add them in another PR.

Reply:
sure, I'd like to add node 8/10.